### ISSUE TITLE
Cannot confirm transaction with a safe with unsupported base contract

### ISSUE THREAD

**Issue by mikhailxyz** [safe-global/web-core#1601]
## Bug description
More info available here: https://github.com/5afe/safe-support/issues/125

## Environment
 - Browser: Chrome
 - Wallet: MetaMask
 - Chain: Ethereum mainnet

## Steps to reproduce
 1. Create a Safe
 2. Use https://github.com/rmeissner/safe-migrator to migrate it to a custom implementation contract
 3. Try to transact

## Expected result
Transaction can be confirmed

## Obtained result
Transaction cannot be confirmed

## Screenshots
![image](https://user-images.githubusercontent.com/16622558/214849542-925cd5d8-ce8f-41e5-8dc5-6173abb35a08.png)

**Comment by Eulinky** [comment]
I got a status update to clarify the current situation:

### TL;DR
The safe has been successfully re-migrated to a (hopefully) supported base contract: `0x3E5c63644E683549055b9Be8653de26E0B4CD36E`
We can create and confirm transactions.
Very gladful for your help!

### Chronological Events and one last question

It all started (after nonce 24) with the problem that we were not able to create a TransferToken transaction using the NEW web app. It constantly showed the skeletons on the Review Transaction screen (see above).

In the github issue Mikhail suggested to use Richards Migration tool to reverse to a supported base contract. I did that and it resulted in a proposed transaction in our Safe queue with nonce 25.

The problem describe above persisted and the other owners were not able to confirm the queued tx.

Mikhail then suggested to use the OLD web app to confirm. When I opened the re-migration tx (nonce 25) I saw 0/3 confirmations but I couldn't confirm - the button was enabled but nothing happened.

So I tried to create a new Transfer Coin tx (nonce 26) and that worked, BUT - it didn't show up in the queue. 

Out of curiosity I created another Transfer Coin tx which didn't show up in the queue as well and that's why I checked the Safe API and I found both transactions having the same nonce on the /multisig-transaction route:

https://safe-client.safe.global/v1/chains/56/safes/0x7c68fC19dE700Af3b4cC9be2ad07A660AC707eff/multisig-transactions

![image](https://user-images.githubusercontent.com/1304814/214888753-be7f89e3-4535-45af-9410-1161aeebd8ba.png)

I checked back with the other two owner later and suprisingly they were able to confirm the re-migration tx (nonce) using the OLD web app. Yihaa! I think we can consider ourselves lucky that it has not yet been taken off line :)

After that I created a new transfer token tx on the NEW web app and this finally worked (thanks to the new base contract). However: the transaction coming up in the queue showed me as _3 hours ago_. It seems creating a the same tx again did not really create new one but made the one from before visible on the queue. Works for me!

There is now only one tx in question: 0xa41c47689ee7b5ac8e1d71c1b1c1200fcc35deb3bf064085b16f9b3bcd14d8a7
It's not required anymore, does not show up in the queue BUT has the same nonce (26) as the one that's currenty in progress. 

I guess the best way is to create the same tx again (to make it visible) and then reject? Just want to re-confirm before I'm breaking something else :)

And thanks again for your help!

**Comment by katspaugh** [comment]
Glad to see the issue was resolved!

Closing because as the notification says, it's expected that unofficial contracts aren't supported.
